CMakeLists.txt

```diff
@@ -1,76 +1,124 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(libyaml_vendor)
+project(libyaml_vendor VERSION 1.0.0)
 
-find_package(ament_cmake REQUIRED)
+option(FORCE_BUILD_VENDOR_PKG
```

Comment:
Please include `YAML` in this option name, as it without it makes it hard to target this package using colcon which passes cmake args to all packages indiscriminately.